botfront/imports/ui/components/nlu/import-export/DataImport.jsx

```diff
@@ -32,12 +32,17 @@ export default class DataImport extends React.Component {
     }
 
     onDrop = (files) => {
+        const { model, instanceHost } = this.props;
         files.forEach((file) => {
             const reader = new FileReader();
-            reader.onload = () => this.setState({ values: reader.result });
+            reader.onload = () => {
```

Comment:
Don't convert to json if its already a json file

Reply:
ok will do it